ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly.yaml

```diff
@@ -842,6 +842,54 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-heterogeneous
     workflow: cucushift-installer-rehearse-gcp-ipi
+- as: gcp-ipi-to-multiarch-mini-perm-f14-compliance
```

Comment:
Please don't use the "mini-perm" profiles. Because the OLM might be disabled in such jobs.